Introduce frontmatter in step 2 (#7704)
jekyll/jekyll: docs/_docs/step-by-step/02-liquid.md

```  
{% endraw %}

To get our changes processed by Jekyll we need to add [front matter](03-front-matter/) to the top of the page:

> {% raw %}

Reviewer: The `raw` tags are only necessary if you have *Liquid constructs* that need to be *escaped*. The following is sufficient:

    ```markdown
    ---
    # Front Matter tells Jekyll to process Liquid
    # and allow converting Markdown into HTML
    ---
    ```

```markdown
---
# front matter tells Jekyll to process Liquid
---
```

> {% endraw %}

Reviewer: Likewise, please remove this closing `endraw` tag as well.

> To get our changes to show up on the page, we need to add something called a `frontmatter` to the top of our HTML. We will discuss frontmatters more in the next section. For now, just add this to the top of your html file:

Reviewer 2: ```suggestion
To get our changes processed by Jekyll we need to add [front matter](03-front-matter/) to the top of the page:
```

> # this is an empty front matter

Reviewer 2: ```suggestion
# front matter tells Jekyll to process Liquid
```

> If you are looking at the rendered file in the browser, you should now see that our "Hello World!" has been downcased.

Reviewer 2: ```suggestion
Our "Hello World!" will now be downcased on render.
```
